sphinx_gallery/tests/test_full.py

```diff
@@ -62,3 +64,11 @@ def test_embed_links(sphinx_app):
     assert 'numpy.arange.html' in lines
     # assert '#module-matplotlib.pyplot' in lines
     # assert 'pyplot.html' in lines
+
+
+def test_clean(sphinx_app):
```

Comment:
In case there is no better suggestions for ensuring that this tests runs last, can you add a comment mentioning that this test should be at the end of the file in order not to interfere with other tests.